src/core/expression/qgsexpressionnode.h

```diff
@@ -252,6 +252,33 @@ class CORE_EXPORT QgsExpressionNode SIP_ABSTRACT
      */
     bool prepare( QgsExpression *parent, const QgsExpressionContext *context );
 
+    /**
```

Comment:
Do you think this needs to be in the stable API? I'd lean toward omitting it from the Python bindings to give us more flexibility later.

Reply:
Don't forget the \since too

Reply:
Yeah that is a good point.